core/dynamo.py

```diff
@@ -7,6 +7,10 @@
 from flask import current_app, g
 import boto3
 # from botocore.exceptions import ClientError
+import logging.config
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
```

Comment:
why? `current_app` is available here. `set_dynamo_table` happens in the `request` lifecylce. shouldn't all this be done already?